mivs/templates/mivs_applications/index.html

```diff
@@ -1,216 +1,222 @@
+          at least 30 seconds of uninterrupted gameplay.  This video will be utilized by our judges to
+          determine if your game is accepted through Round 1 of judging.  If your game is accepted into
+          the showcase, this video link may be embedded on the MIVS web-page and/or other places that
+          MAGFest advertises the MIVS area.  You must submit a video link before {{ c.ROUND_ONE_DEADLINE|datetime_local }}.
+        {% elif c.AFTER_ROUND_ONE_DEADLINE and not c.HAS_INDIE_ADMIN_ACCESS %}
+          You may no longer submit this game for consideration, as it is past the
+          {{ c.ROUND_ONE_DEADLINE|datetime_local }} deadline.
+        {% else %}
+          Once you're ready for our judges to look at the video link you've provided, you can submit it
+          to our panel of judges for consideration anytime until {{ c.ROUND_ONE_DEADLINE|datetime_local }}:
+          <form method="post" action="submit_video">
+            {{ csrf_token() }}
+            <input type="hidden" name="id" value="{{ game.id }}" />
+            <input class="btn btn-primary" type="submit" value="Submit Video for Consideration" />
+          </form>
+        {% endif %}
+
+        {% if c.ALLOW_GAME_SUBMISSION and game.status != c.VIDEO_DECLINED %}
+          <h4>Game Submission</h4>
+          {% if game.submitted %}
+            This game has been submitted.
+          {% elif needs_screenshots %}
+            Please upload at least two screenshots before submitting your game for consideration.
+          {% elif c.AFTER_ROUND_TWO_DEADLINE and not c.HAS_INDIE_ADMIN_ACCESS %}
+            You may no longer submit this game for consideration, as it is past the
+            {{ c.ROUND_TWO_DEADLINE|datetime_local }} deadline.
+          {% elif not game.submittable %}
+            You cannot submit this game until the following criteria are met:
+            <ul>
+              {% for step in game.missing_steps %}
+                <li>{{ step }}</li>
+              {% endfor %}
+            </ul>
+          {% else %}
+            You've met all the prerequisites to submit this game for consideration.  After submitting you may no longer
+            edit the game details unless the judges kick the game back to you and request edits.  You will still be able
+            to manage screenshots and the game build information, such as build status and link, after submitting.  Our
+            judges will not look at your game until you officially submit it by clicking this button:
+            <form method="post" action="submit_game" class="form-horizontal">
+              {{ csrf_token() }}
+              <input type="hidden" name="id" value="{{ game.id }}" />
+              <input class="btn btn-primary" type="submit" value="Submit Game For Consideration" />
+            </form>
+          {% endif %}
+        {% endif %}
+
+        {% if c.BEFORE_ROUND_TWO_DEADLINE or c.HAS_INDIE_ADMIN_ACCESS %}
```

Comment:
Whoops, this diff is terrible because I fixed the whitespace in it. Basically this is the only block that's actually changed. I replaced the `game.link_to_game` check with this one, allowing studios to change the game's build information any time before the deadline. Then I added the form you see below, so they can change ANY of the build information, not just the game codes.